### PR TITLE
 branch conflicts have resolved

### PR DESCRIPTION
**Pull request by AndriiK139MCAF**
Brunch with resolved conflict

### CODE REVIEW

File: mo_hr_timesheets/models/custome_contact_fields.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class CustomContactFields(models.Model):
+    """Adding custom address
+       fields"""
+    _inherit = 'hr.employee'
+
+    street_employe = fields.Char(string=' Street ', help='Street address')
+    city_employe = fields.Char(string=' City ', help='City')
+    state_id_employe = fields.Many2one(
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/custome_contact_fields.py:11]: rename  to state_employee_id

File: mo_hr_timesheets/models/custome_contact_fields.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class CustomContactFields(models.Model):
+    """Adding custom address
+       fields"""
+    _inherit = 'hr.employee'
+
+    street_employe = fields.Char(string=' Street ', help='Street address')
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/custome_contact_fields.py:9]: rename street_employe to street_employee

File: mo_hr_timesheets/models/custome_contact_fields.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    city_employe = fields.Char(string=' City ', help='City')
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/custome_contact_fields.py:10]: rename city_employe to city_employee

File: mo_hr_timesheets/models/custome_contact_fields.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    state_id_employe = fields.Many2one(
+        'res.country.state',
+        string='Province',
+        default=lambda self: self.env['res.country.state'].search(
+            [('code', '=', 'NB')], limit=1),
+        help='Select the state for the contact.'
+    )
+    country_id_employe = fields.Many2one(
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/custome_contact_fields.py:18]: rename to country_employee_id

File: mo_hr_timesheets/models/custome_contact_fields.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        'res.country',
+        string='Country',
+        default=lambda self: self.env['res.country'].search(
+            [('code', '=', 'CA')], limit=1),
+        help='Select the country for the contact.'
+    )
+    zip_code_employe = fields.Char(string=' Zip Code ', help='Zip Code')
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/custome_contact_fields.py:25]: rename to zip_code_employee

File: mo_hr_timesheets/views/employee_information.xml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='personal_information']" position="attributes">
+                    <attribute name="string">Personal Information</attribute>
+            </xpath>
+            <xpath expr="//field[@name='address_home_id']" position="after">
+                <field name="street_employe"/>
+                <field name="city_employe"/>
+                <field name="state_id_employe"
+                       options="{'no_open' : True}"/>
+                <field name="zip_code_employe"/>
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/views/employee_information.xml:18]: rename to zip_code_employee

File: mo_hr_timesheets/views/employee_information.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+                <field name="country_id_employe"/>
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/views/employee_information.xml:19]: rename to country_employee_id

File: mo_hr_timesheets/views/employee_information.xml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="priority">10</field>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='personal_information']" position="attributes">
+                    <attribute name="string">Personal Information</attribute>
+            </xpath>
+            <xpath expr="//field[@name='address_home_id']" position="after">
+                <field name="street_employe"/>
+                <field name="city_employe"/>
+                <field name="state_id_employe"
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/views/employee_information.xml:16]: rename to  state_employee_id

**Comment by AndriiK139MCAF** [comment]
fix names issue

File: mo_hr_timesheets/views/employee_information.xml (hunk 8 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/views/employee_information.xml:16]: fix

File: mo_hr_timesheets/views/employee_information.xml (hunk 7 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/views/employee_information.xml:19]: fix

File: mo_hr_timesheets/views/employee_information.xml (hunk 6 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/views/employee_information.xml:18]: fix

File: mo_hr_timesheets/models/custome_contact_fields.py (hunk 5 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/models/custome_contact_fields.py:25]: fix

File: mo_hr_timesheets/models/custome_contact_fields.py (hunk 4 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/models/custome_contact_fields.py:18]: fix

**Pull request closed without merging**